src/lib/coda_networking/coda_networking.ml

```diff
@@ -460,6 +451,15 @@ module Make (Inputs : Inputs_intf) = struct
     |> Deferred.ignore |> don't_wait_for ;
     online_reader
 
+  let wrap_rpc_data_in_envelope conn data =
+    let inet_addr = Unix.Inet_addr.of_string conn.Host_and_port.host in
+    let sender =
+      if Unix.Inet_addr.equal inet_addr Unix.Inet_addr.localhost then
+        Envelope.Sender.Local
```

Comment:
This has the effect of stopping bans from happening at all in our integration tests. I'd much rather have an envelope with localhost as the sender